projects/Mallard/src/services/remote-config.ts

```diff
@@ -14,7 +14,7 @@ const remoteConfigDefaults = {
     logging_enabled: true,
     join_beta_button_enabled: false,
     lightbox_enabled: true,
-    generate_share_url: false,
+    generate_share_url: true,
```

Comment:
the remote switch is just in case there's some unexpected problem with the javascript it will be good to be able to switch it off, but it should be on by default

Reply:
I like this